code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -620,3 +620,36 @@
 /obj/effect/mob_spawn/human/gamer/Destroy()
 	new/obj/structure/showcase/machinery/oldpod/used(drop_location())
 	return ..()
+
+
+/obj/effect/mob_spawn/human/construction_worker
+	name = "construction worker sleeper"
+	desc = "A humming cryo pod."
+	mob_name = "a construction worker"
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	mob_species = /datum/species/human
+	short_desc = "Nanotrasen has hired you to construct a new station, as a backup in the case that ss13 is no longer livable"
```

Comment:
Nanotrasen has hired you to construct a new station, as a backup in the case that SS13 is no longer liveable.